crawler/twitterCrawlers.py

```diff
@@ -37,43 +37,39 @@ def __connect_to_db(self):
                 self.__log_to_file(self.name + " connected to the database.")
                 return driver
             except Exception as exc:
-                self.__log_to_file("0 - ERROR -> %s. Crawler could not connect to the database. Retrying in 5s..." % exc)
+                self.__log_to_file(
+                    "0 - ERROR -> %s. Crawler could not connect to the database. Retrying in 5s..." % exc)
                 time.sleep(5)
                 continue
 
     def run(self):
```

Comment:
all the previous code here was just necessary to log when the queue is empty.
You could also write: 
```python
try:
    user_id = self.q.get_nowait()
    self.crawl(user_id)
except: multiprocessing.queues.Empty:
    self.__log_to_file("blabla")
    time.sleep(5)
```

but i decided to not write as many extra lines just for a simple log.